manager2/src/app/user/user.component.html

```diff
@@ -173,11 +173,17 @@ <h4>Information</h4>
                               </label>
                           </div>
                           <div class="form-check row">
-                                <input [ngModelOptions]="{standalone: true}" [(ngModel)]="user.is_fake" type="checkbox" class="form-check-input">
+                              <input [ngModelOptions]="{standalone: true}" [(ngModel)]="user.is_fake" type="checkbox" class="form-check-input">
                               <label class="col-6 form-check-label">
                                   Fake/App account
                               </label>
                           </div>
+                          <div class="form-check row">
+                              <input [ngModelOptions]="{standalone: true}" [(ngModel)]="user.never_expire" type="checkbox" class="form-check-input">
```

Comment:
If undefined (for all existing users), won t it fail?

On user.get should we not set never_expire to false?

Reply:
it work on my test, but I add it to get method to avoid any potential issue e47b318ecf6de6bc1c782906061df0b124589c6c